engineapi/instance_manager.go

```diff
@@ -784,3 +784,27 @@ func (c *InstanceManagerClient) VersionGet() (int, int, int, int, error) {
 	return output.InstanceManagerAPIMinVersion, output.InstanceManagerAPIVersion,
 		output.InstanceManagerProxyAPIMinVersion, output.InstanceManagerProxyAPIVersion, nil
 }
+
+func (c *InstanceManagerClient) LogSetLevel(dataEngine longhorn.DataEngineType, component, level string) error {
+	if err := CheckInstanceManagerCompatibility(c.apiMinVersion, c.apiVersion); err != nil {
+		return err
+	}
+
+	if c.GetAPIVersion() < 6 {
```

Comment:
Can we use `CurrentInstanceManagerAPIVersion` instead of a primitive version directly? Should we update `CurrentInstanceManagerAPIVersion` to 6 as well?

Reply:
Using `CurrentInstanceManagerAPIVersion` looks good.

> Should we update CurrentInstanceManagerAPIVersion to 6 as well?

Yes, because we update the IM version to 6.